### PR TITLE
Check for .nomedia file inside tv show folders

### PR DESCRIPTION
**Pull request by ThiefMaster**
Previousy, only the base source folder was checked for a .nomedia file, which was rather pointless since you'd simply not add a source if you didn't want it in your library.

Now the actual TV show folders are also checked, so you can add a `.nomedia` file inside the folder of a TV show to exclude it from being added to the library.

Related forum post where I reported the issue: http://forum.kodi.tv/showthread.php?tid=251714

### CODE REVIEW

**Comment by NedScott** [comment]
I'm almost certain that this is intentional behavior, that .nomedia is not meant to be reclusive. I could be wrong, but it makes a lot of sense for people who use sub-folders as a means of organization, while preventing the "organization folder" from being scanned as an actual show.

**Comment by ThiefMaster** [comment]
http://trac.kodi.tv/ticket/16444

File: xbmc/video/VideoInfoScanner.cpp [hunk 1]
```diff
@@ -511,6 +511,11 @@ namespace VIDEO
 
   INFO_RET CVideoInfoScanner::RetrieveInfoForTvShow(CFileItem *pItem, bool bDirNames, ScraperPtr &info2, bool useLocal, CScraperUrl* pURL, bool fetchEpisodes, CGUIDialogProgress* pDlgProgress)
   {
+    if (pItem->m_bIsFolder && IsExcluded(pItem->GetPath()))
+    {
+      CLog::Log(LOGWARNING, "Skipping show '%s' with '.nomedia' file in its directory, it won't be added to the library.", CURL::GetRedacted(pItem->GetPath()).c_str());
```
Comment (Razzeee) [inline xbmc/video/VideoInfoScanner.cpp]: Not sure if this should be a LOGWARNING, seems more like an Info if not even Debug?

Comment (ThiefMaster) [inline xbmc/video/VideoInfoScanner.cpp]: I used the one that was also used for movie .nomedia fiels

Comment (Razzeee) [inline xbmc/video/VideoInfoScanner.cpp]: NVM then. If we want to change that, we will do it in another PR.

**Comment by Razzeee** [comment]
I'll merge this tomorrow, if nobody says anything.

jenkins build this please

**Comment by ThiefMaster** [comment]
hrm, github shows failed CI checks but the details link goes to a 404

**Comment by anaconda** [comment]
I'm pretty sure the build error was not related to this PR, but let's see.
jenkins build this please

**Comment by ThiefMaster** [comment]
ping, any news about this?

**Comment by MartijnKaijser** [comment]
@Razzeee why was this put in Jarvis branch? :)

**Comment by Razzeee** [comment]
Crap. Probably because github doesn't show the target branch in the mobile layout :(

**Comment by Razzeee** [comment]
@ThiefMaster can you please do another PR against master?

**Comment by ThiefMaster** [comment]
#9170

**Comment by Razzeee** [comment]
Wiki is already correct.
